pages/userSettings/userSettings.ejs

```diff
@@ -13,6 +13,59 @@
     <div id="content" class="container">
       <h1 class="mb-4">Settings</h1>
 
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex align-items-center">
+          Logged-in user profile information
+          <a href="/pl/logout" class="btn btn-danger btn-sm ml-auto">Logout</a>
+        </div>
+          <table class="table table-sm two-column-description">
+            <tbody>
+              <tr>
+                <th>User ID (UID)</th>
+                <td>
+                  <%= authn_user.uid %>
+                </td>
+              </tr>
+              <tr>
+                <th>User Name</th>
+                <td><%= authn_user.name %></td>
+              </tr>
+              <tr>
+                <th>Unique Id Number (UIN)</th>
+                <td><%= authn_user.uin %></td>
+              </tr>
+              <tr>
+                <th>Institution</th>
+                <td><%= authn_institution.long_name %> (<%= authn_institution.short_name %>)</td>
+              </tr>
+              <tr>
+                <th>Authentication method</th>
+                <td>
+                  <%= authn_provider_name %>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+      </div>
+
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex">
+          Browser configuration
+        </div>
+        <div class="card-body">
+          <p>These options will let you reset some browser settings related to technology inside PrairieLearn.
+            Use these buttons to reset things if things are broken for you.
+          </p>
+          <p>
+          <button class="btn btn-md btn-info"
+            onClick="localStorage.removeItem('MathJax-Menu-Settings');alert('MathJax menu settings have been reset');">
+            Reset MathJax menu settings
+          </button> Can be helpful if your math shows up as
+          <strong>$ x = \frac {-b \pm \sqrt {b^2 - 4ac}}{2a} $</strong>
```

Comment:
Let's move the explanatory text into a `<p>` that precedes the button. Given that there's only one button, I'd be inclined to just describe the option instead of referring to "these options". Maybe copy like this?

> Resetting the MathJax settings may help if math formulas are showing as [bad example here].

Reply:
I tried it a few ways and I feel the left justified button works best.

Reply:
"precedes" as in on the preceding line as opposed to on the same line as the button; I agree that a left-justified button is the way to go.

Reply:
![image](https://user-images.githubusercontent.com/10765199/222856061-c73977a7-6c13-41e2-93ce-7d59cb10bada.png)

I'm OK with putting the text on the line above the button. That degrades better with long text than inline with the button.